changelog.d/10084.feature

```diff
@@ -0,0 +1 @@
+Add support for routing more requests to workers.
```

Comment:
I do wonder if we should do a better job of helping people upgrade their reverse proxies by giving more info here. 🤷

Reply:
Probably :shrug: